src/pages/ConnectedAppsSettings/ConnectedAppsSettings.tsx

```diff
@@ -17,6 +34,44 @@ export const ConnectedAppsSettings: React.FC = React.memo(() => {
     }),
   );
 
+  const services: Service[] = [
+    {
+      title: 'Spotify',
+      icon: <ServiceIconSpotify />,
+      token: user?.spotifyOAUTH,
+      register: registerSpotify,
+      remove: async () => removeSpotify(user!.id),
+    },
+    {
+      title: 'Yandex Music',
+      icon: <ServiceIconYandex />,
+      token: user?.yandexOAUTH,
+      register: () => {},
+      remove: async () => removeSpotify(user!.id),
+    },
+  ];
+
+  const enableModal = (service: Service) => {
+    setCurrentService(service);
+    setIsModalActive(true);
+  };
+
+  const disableModal = async () => {
+    setIsLoading(true);
+
+    await currentService
+      ?.remove()
+      .then(() => {
+        setIsModalActive(false);
+      })
+      .catch(() => {
```

Comment:
У тебя вот тут бек будет присылать ошибки. Введи неправильный id юзера на 43 строке, и попробуй посмотреть через лог результат error
```suggestion
      .catch((error) => {
```